test/unit/spec/appWidgets/validationSpec.js

```diff
@@ -85,324 +109,340 @@ define([
+                    expect(Validation.validateTrue(value)).toEqual({
+                        isValid: true,
+                        errorMessage: null,
+                        diagnosis: Constants.DIAGNOSIS.VALID,
+                        value: value,
+                        parsedValue: value,
+                    });
+                });
             });
         });
 
-        it('validateTextString - Validate a simple string without constraints', () => {
-            expect(Validation.validateTextString('test', {}).isValid).toEqual(true);
-        });
+        // STRING
+        describe('validateTextString', () => {
+            it('a simple string without constraints', () => {
+                expect(Validation.validateTextString('test', {}).isValid).toEqual(true);
+            });
 
-        it('validateTextString - Validate a simple string required and supplied', () => {
-            const result = Validation.validateTextString('test', {
-                required: true,
+            it('a simple string required and supplied', () => {
+                const result = Validation.validateTextString('test', {
+                    required: true,
+                });
+                expect(result.isValid).toEqual(true);
             });
-            expect(result.isValid).toEqual(true);
-        });
-        it('validateTextString - Validate a simple string, required, empty string', () => {
-            const result = Validation.validateTextString('', { required: true });
-            expect(result.isValid).toEqual(false);
-        });
-        it('validateTextString - Validate a simple string, required, null', () => {
-            const result = Validation.validateTextString(null, {
-                required: true,
+            it('a simple string, required, empty string', () => {
+                const result = Validation.validateTextString('', { required: true });
+                expect(result.isValid).toEqual(false);
             });
-            expect(result.isValid).toEqual(false);
-        });
-        it('validateTextString - Validate a simple string, min and max length, within range', () => {
-            const result = Validation.validateTextString('hello', {
-                required: true,
-                min_length: 5,
-                max_length: 10,
+            it('a simple string, required, null', () => {
+                const result = Validation.validateTextString(null, {
+                    required: true,
+                });
+                expect(result.isValid).toEqual(false);
             });
-            expect(result.isValid).toEqual(true);
-        });
-        it('validateTextString - Validate a simple string, min and max length, below', () => {
-            const result = Validation.validateTextString('hi', {
-                required: true,
-                min_length: 5,
-                max_length: 10,
+            it('a simple string, min and max length, within range', () => {
```

Comment:
git's diff algorithm has gone a bit wonky here -- there are no changes to the test content but the title and indent change has thrown it for a loop.